### ISSUE TITLE
Is there any plan to support Loupedeck devices?

### ISSUE THREAD

**Issue by madmalkav** [StreamController/StreamController#223]
Loupedeck provides devices similar to Streamdeck. Is there any plan yo support those?

**Comment by Core447** [comment]
Not really, but using [this](https://github.com/devleaks/python-loupedeck-live) library implementing at least support for the touch button grid should be relatively easy.
